auctioneerrpc/auctioneer.proto

```diff
@@ -1,8 +1,8 @@
 syntax = "proto3";
 
-package poolrpc;
+package auctioneerrpc;
```

Comment:
is this a breaking change?

Reply:
Argh, yes it is... I think we'll need to serve everything under both paths for a while. Same proto, different gRPC package. Good catch, totally forgot about the package being in the URI.

Reply:
Breaking change from an RPC PoV since this module name is used in the path to the actual HTTP endpoint? 

What do we need to do in order to serve both paths on the server end of things?

Reply:
Okay, I figured that we can just go ahead and move the _golang_ package and use it under the `auctioneerrpc` package name but keep the RPC name as it was. That way this is non breaking as on the RPC level nothing changes.
I confirmed that this doesn't break old clients.